cmd/bbolt/command_check.go

```diff
@@ -9,16 +9,19 @@ import (
 	"go.etcd.io/bbolt/internal/guts_cli"
 )
 
+var pageId uint64
+
 func newCheckCommand() *cobra.Command {
 	checkCmd := &cobra.Command{
 		Use:   "check <bbolt-file>",
 		Short: "verify integrity of bbolt database data",
-		Args:  cobra.ExactArgs(1),
+		Args:  cobra.MinimumNArgs(1),
 		RunE: func(cmd *cobra.Command, args []string) error {
 			return checkFunc(cmd, args[0])
 		},
 	}
 
+	checkCmd.Flags().Uint64VarP(&pageId, "pageId", "", pageId, "page Id")
```

Comment:
Maybe we can enhance the help to explain that if a page id is specified, it will check the integrity starting from that page.

Reply:
done 👍🏽